src/templates/static/compliance/generateReport.yml

```diff
@@ -0,0 +1,31 @@
+- name: generate Compliance report
+  hosts: "@@HOSTS@@"
+  become: true
+  gather_facts: false
+  vars:
+    insights_signature_exclude: /hosts,/vars/insights_signature
+    insights_signature: !!binary |
```

Comment:
I added a fake signature... Wasn't sure if I should just leave this field blank but figured it wouldn't matter since this will be updated when the content is signed 🤔

Reply:
:slightly_smiling_face: rather leave it blank.